demo/demo-schema/src/main/java/org/apache/servicecomb/demo/RestObjectMapperWithStringMapper.java

```diff
@@ -43,6 +45,8 @@
  *  conversion change. You must write  convertValue to check possible types using.
  */
 public class RestObjectMapperWithStringMapper extends RestObjectMapper {
```

Comment:
put this demonstrate class in demo project or treat it as a product class?

Reply:
This is a demo